app/controllers/concerns/workflow/controller.rb

```diff
@@ -86,6 +84,19 @@ def send_transition_email
       end
     end
 
+    # Who should receive emails about changes to a `project`s workflow position.
+    # In lieu of a subscription based model where users can manage their own desired status updates.
+    # NOTE: yuk.
+    def notifiable_users
+      # Does the user performing the transition need notifying?
+      scope = @project.users.where.not(id: current_user)
+
+      # Copied from original project rejected notification...
+      scope = scope.where.not(id: @project.users.odr_users) if @project.application?
```

Comment:
would this be notifying ODR for EOI, MBIS, CAS & other future application types?

Reply:
"ODR" as in the role, or as in the collective team of people?

Hypothetically speaking, if a given user has some kind of `Grant` on the `project`, and there is an appropriate method on the mailer that would handle the required mail out, then they would be mailed. 

This is one of the reasons why the generic mail method (`transitioned`) is *not* currently targetable via the `state_changed` mail method; we're not looking to spam users with every state change. 

Under the current configuration and available mail methods for state changes, the only mailing that will happen via this route is for projects of any kind being rejected, which is no different from what is currently happening via `Project#status_change_notifier` and `Project#odr_rejected_notification` as far as I can tell.

The exception for ODR users + applications, was bourne out of one user not wanting to receive an email at a specific point (a very specific edge case, which is simply being reimplemented here). We can add/tweak as needed but should be mindful that this is a smell that will probably grow over time.

This really feeds into the idea of having users manage the types of emails/notifications they are interested in receiving being a potentially beneficial bit of development; it empowers users to tune into what they want, and saves us having to bloat the codebase with conditional logic to cover edge case after edge case.

Reply:
Yep, probably.

I'm sure they're similarly not interested in the cases highlighted in your initial comment, but the ticket that asked for an exception didn't specify those, so they got exactly what was asked for 😉